_includes/head.html

```diff
@@ -1,13 +1,16 @@
 <meta charset="utf-8">
 <meta http-equiv="x-ua-compatible" content="ie=edge">
-<title>Not 911 - {{ page.title }}</title>
+<title>{{ page.title }} | Not 911</title>
```

Comment:
Huh, why this change?

Reply:
Browser tabs only have so much space and putting the site name first before the page title makes the page title get cut off. The favicon already gives the context that this is the not 911 tab so there's no need to put the site title first.

Most websites put the page title first too, so I think it was just us not thinking closely about this when it was first implemented.